doc_source/cni-custom-network.md

```diff
@@ -10,6 +10,22 @@ The use cases discussed in this topic require [Amazon VPC CNI plugin for Kuberne
 
 Enabling this feature effectively removes an available elastic network interface \(and all of its available IP addresses for pods\) from each worker node that uses it\. The primary network interface for the worker node is not used for pod placement when this feature is enabled\. You should choose larger instance types with more available elastic network interfaces if you choose to enable this feature\.
 
+Once you enable custom network and apply ENIConfig in different subnet or VPC CIDR, please note that the primary elastic network interface's cannot be used to provide Pod IP addresses. If you are using [Amazon EKS-Optimized Linux AMI](https://docs.aws.amazon.com/en_us/eks/latest/userguide/eks-optimized-ami.html), it is necessary to reduce the max number of Pods that can be scheduled on Node as defined in the [eni-max-pods.txt](https://github.com/awslabs/amazon-eks-ami/blob/master/files/eni-max-pods.txt) config file and restart your worker node manually. This change will let kubelet running on worker node apply new `--max-pods` configuration. For more information, see [kubelet reference](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) in the *Kuberentes documentation*.
+
+The following formula explained how to get the number of Pod can be scheduled on your worker node after you applying the configuration:
```

Comment:
quick grammatical fixups:

s/explained/explains/
s/of Pod can be scheduled/of Pods that can be scheduled/
s/after you applying/after you apply/